deployer/conf/elasticsearch-logging.yml

```diff
@@ -1,16 +1,25 @@
 # you can override this using by setting a system property, for example -Des.logger.level=DEBUG
 es.logger.level: INFO
-rootLogger: ${es.logger.level}, console
+rootLogger: ${es.logger.level}, file
 logger:
   # log action execution errors for easier debugging
   action: WARN
 
+  # deprecation logging, turn to DEBUG to see them
+  deprecation: WARN, deprecation_log_file
+
   # reduce the logging for aws, too much is logged under the default INFO
   com.amazonaws: WARN
 
   io.fabric8.elasticsearch: ${PLUGIN_LOGLEVEL}
   io.fabric8.kubernetes: ${PLUGIN_LOGLEVEL}
 
+  # aws will try to do some sketchy JMX stuff, but its not needed.
+  com.amazonaws.jmx.SdkMBeanRegistrySupport: ERROR
+  com.amazonaws.metrics.AwsSdkMetrics: ERROR
```

Comment:
@portante Why do we need this? Isn't this relevant to Elastic AWS plugin only? (https://stackoverflow.com/a/35752768)